client/src/containers/filters/years-range/component.tsx

```diff
@@ -148,23 +130,21 @@ export const YearsRangeFilter: React.FC<YearsRangeFilterProps> = ({
                   loading={loading}
                   options={startYearOptions}
                   value={startYearOption}
-                  onChange={({ value }) => {
-                    onChange?.({ startYear: Number(value), endYear });
-                  }}
+                  onChange={({ value }) => onChange?.({ startYear: value, endYear })}
                   placeholder={placeholderFrom}
                 />
                 <div>To</div>
                 <Select<number>
                   loading={loading}
                   options={endYearOptions.map((option) => ({
```

Comment:
Can we move the options to a `useMemo` hook, please? So we avoid mapping them in every render.